packages/2018-disaster-resilience/src/components/App/index.js

```diff
@@ -49,7 +58,9 @@ const App = () => (
     </p>
     {/* <IncreasingSocialCapital /> */}
     <PullQuote quoteText="The #1 thing you can do to increase social capital is to meet your neighbors. Do you know 3 people within a 3 block radius of your house?" />
-    <ProactivePlanning />
+    <section className={sectionMarginMedium}>
```

Comment:
should this be `css` and not `className`?

Reply:
🙀YES